src/Configuration.php

```diff
@@ -121,7 +126,21 @@ public function getPath()
      */
     public function disableColors()
     {
-        return $this->write('colorsEnabled', false);
+        if ( $this->colorsEnableExplicit ) {
+            return $this;
+        }
+        else {
```

Comment:
can we remove the else here? the return statement of the preceding `if` should make sure the code in the `else` clause is executed